pkg/beacon/relay/entry/submission.go

```diff
@@ -26,6 +26,8 @@ func (res *relayEntrySubmitter) submitRelayEntry(
 	newEntry []byte,
 	groupPublicKey []byte,
 	startBlockHeight uint64,
+	relayEntrySubmittedChannel <-chan uint64,
+	relayEntryTimeoutChannel <-chan uint64,
```

Comment:
I think we should revert the change in this file. The messages from those channels can be read only one time. Either here or in `submitRelayEntry` function calling this one. 

I think we should be fine with the previous approach with just `onSubmittedResultChan`. When we are here, there is no really a possibility for a timeout. Submission is a non-blocking call, we submit to the pool, we don't wait for the transaction to be mined.

Reply:
Yep, but it is not a problem in the context we use those channels here. If any success/timeout signal arrives when we are in the message loop we will exit before reaching the submitter. If something arrives after the message loop, the submitter will nicely handle it.

Reply:
Ah, it makes sense. Can we add a comment about it to `entry.go` where we call `submitter.submitRelayEntry(` ?